lib/v2/gzdaily/router.js

```diff
@@ -0,0 +1,4 @@
+module.exports = (router) => {
+    router.get('/app/:column', require('./app'));
+    router.get('/cmc/:channel', require('./cmc'));
```

Comment:
```suggestion
    router.get('/app/:column?', require('./app'));
    router.get('/cmc/:channel?', require('./cmc'));
```
https://github.com/DIYgod/RSSHub/blob/973de743459da7038604ac9d1bdbfbbcd0d414fb/lib/v2/gzdaily/app.js#L9
https://github.com/DIYgod/RSSHub/blob/973de743459da7038604ac9d1bdbfbbcd0d414fb/lib/v2/gzdaily/cmc.js#L9